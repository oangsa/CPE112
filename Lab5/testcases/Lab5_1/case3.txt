Enqueue 36
Enqueue 38
Dequeue
Dequeue
END
