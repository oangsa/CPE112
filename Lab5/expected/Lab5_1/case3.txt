36 
38 
Queue is empty
